logistics_planning_base/models/logistics_schedule.py

```diff
@@ -0,0 +1,76 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields
+
+TRANSPORT_TYPE = [
+    ('ground', 'Ground'),
+    ('ocean-going', 'Ocean-Going'),
+]
+
+class LogisticsSchedule(models.Model):
+    _name = 'logistics.schedule'
+    _description = 'logistics.schedule'
+
+    name = fields.Char(compute='_compute_name')
+    company_id = fields.Many2one('res.company', string='Company', default=lambda self: self.env.company)
+    state = fields.Selection(
+        selection=[
+            ('draft', 'Draft'),
+            ('ready', 'Ready'),
+            ('done', 'Done'),
+            ('cancel', 'Cancel'),
+        ]
+    )
+    type = fields.Selection(
+        selection=[
+            ('input', 'Input'),
+            ('output', 'Output'),
+        ]
+    )
+    stock_move_id = fields.Many2one('stock.move')
+    picking_id = fields.Many2one('stock.picking', related='stock_move_id.picking_id')
+    destination_partner_id = fields.Many2one('res.partner')
+    partner_id = fields.Many2one('res.partner')
+    transport_type = fields.Selection(
+        selection=TRANSPORT_TYPE
+    )
+    product_id = fields.Many2one('product.product')
+    product_uom = fields.Many2one('uom.uom')
+    product_uom_qty = fields.Float()
+    scheduled_load_date = fields.Date()
+    commitment_date = fields.Datetime()
+    logistic_price_unit_type = fields.Selection(
+        selection=[
+            ('trip', 'Trip'),
+            ('unit', 'Unit'),
+        ]
+    )
+    logistic_price_unit = fields.Float(digits='Product Unit of Measure')
```

Comment:
Wrong digits for a currency field (see e.g. https://github.com/odoo/odoo/blob/13.0/addons/purchase/models/purchase.py#L482). In fact, I also miss `currency_id` field